dali/operators/math/expressions/arithmetic.h

```diff
@@ -115,8 +115,8 @@ DLL_PUBLIC DALIDataType PropagateTypes(ExprNode &expr, const workspace_t<Backend
   }
   auto &func = dynamic_cast<ExprFunc &>(expr);
   int subexpression_count = func.GetSubexpressionCount();
-  DALI_ENFORCE(subexpression_count == 1 || subexpression_count == 2,
-               "Only unary and binary expressions are supported");
+  DALI_ENFORCE(1 <= subexpression_count && subexpression_count <= kMaxArity,
```

Comment:
```suggestion
  DALI_ENFORCE(0 < subexpression_count && subexpression_count <= kMaxArity,
```
To be consistent with L340.

Reply:
Done